common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -95,6 +95,8 @@ object FaciaWidths {
   }
 }
 
+
+//TODO: We should only have one facia widths object. Consolidate this garnett one with the one above.
```

Comment:
I had to do this because the image in immersive cards had a different aspect ratio (in the initial design I was shown). The quality was too low because the size of the image is constrained to the card's width, whereas in the new design the image is supposed to cover the card completely. Cheating with the card width was an easy solution, but there probably is a nicer way of doing this, by adding a new map specific to immersive cards I guess.

Reply:
I think that makes sense. I will take a look in a separate PR :)

Reply:
Yeah, proper HiDPI support for these would be super-cool!